src/ui/server/helpers/get-chosen-ordnance-survey-address/by-index/index.ts

```diff
@@ -0,0 +1,31 @@
+import { MappedOrdnanceSurveyAddress } from '../../../../types';
+
+interface GetChosenOrdnanceSurveyAddressByIndexParams {
```

Comment:
Should this be declared somewhere in a centralised interface collection file?

Reply:
Yes - this PR is a WIP, need another PR merged first.